Sources/TuistCore/NodeLoaders/XCFrameworkLoader.swift

```diff
@@ -50,12 +51,17 @@ public final class XCFrameworkLoader: XCFrameworkLoading {
         self.fileSystem = fileSystem
     }
 
-    public func load(path: AbsolutePath, status: LinkingStatus) async throws -> GraphDependency {
+    public func load(
+        path: AbsolutePath,
+        originalSignature: XCFrameworkOriginalSignatureType,
+        status: LinkingStatus
+    ) async throws -> GraphDependency {
```

Comment:
As mentioned above, I wouldn't touch the framework loader and instead do the signature verification in a standalone mapper.